html/settings/_settings.scss

```diff
@@ -2060,10 +2060,26 @@ $sprk-big-nav-link-font-weight: 400 !default;
 /// The padding of the big navigation
 /// link in the Masthead Extended.
 $sprk-big-nav-link-padding: 0 $sprk-space-s !default;
+/// The underline color of
+/// the active big navigation item in the Masthead Extended.
+$sprk-big-nav-active-color: $sprk-red !default;
 /// The line height
 /// of the big navigation items
 /// in the Masthead Extended.
 $sprk-big-nav-item-line-height: 53px !default;
+/// The active border-bottom
+/// of the big navigation items
+/// in the Masthead Extended.
+$sprk-big-nav-item-active-border-bottom: 0.125rem solid #c8102e !default;
```

Comment:
Is #c8102e a color var or where did it come from?

Reply:
Ope. Should be $sprk-red. Updating.